Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SortType.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities
+{
+    /// <summary>
+    /// Sorting type for collections
+    /// </summary>
+    public enum SortType
```

Comment:
This is too generic of a name. "SortType" seems like it should have entries for QuickSort, BubbleSort, MergeSort etc. This should be something like:
3DSortType
SpatialSortType
VolumetricSortType
PositionalSortingType
GameObjectSortType

Reply:
How About CollationOrderType. Technically this isn't for sorting - you need an order in which to sort.

Reply:
That feels like a synonym for "sort" to me? And it doesn't actually clarify what these are used for. =/

Maybe we're barking at the wrong tree by calling it a sort. Should it be something like SpatialLayoutType instead? SpatialPatternType? 3DLayoutType?

Reply:
This actually has nothing to do with spatial layout. I'd suggest sticking with Collation here as it has to do with naming of objects vs when they appear in the hierarchy, conceptually it makes sense.